### DISCUSSION TITLE
[Piano Notes] (No JSON) Bonetrousle (Undertale)

### DISCUSSION THREAD

**Discussion by Abc2557** [Monika-After-Story/MonikaModDev#7799] [general - piano notes]
Nyehehe!~ Good luck making the JSON for this awesome song, human!
(Ah, yes, another Undertale song)

T U O ] O 9 O P O U T 
W 3 E R T E Q
P [ ] [ P
T U O ] ] [ P [ ] P I Y 
7 U 9 O U T 
E R T R E

I'll probably edit this because it kinda changes keys in some places even though it's not supposed to. 
Anyways, good luck to whoever will make the JSON!

**Comment by satoshi-kun** [comment]
Bonetrousle*

**Comment by Abc2557** [comment]
I've never been able to spell it correctly haha ^^;
